.github/renovate.json5

```diff
@@ -41,6 +41,13 @@
       description: "Disable PRs updating GitHub runners (e.g. 'runs-on: macos-14')",
       enabled: false,
     },
+    {
+      // Disable updates of `zip-rs`; intentionally pinned for now due to ownership change
+      // See: https://github.com/astral-sh/uv/issues/3642
+      matchPackagePatterns: ["zip"],
```

Comment:
Can you add a note on that issue to mention that `red_knot` now also uses the `zip` crate. Or maybe this is something we can discuss in today's standup? The main concern seems to be with 0.66 and 1 but we're now at version 2 already.

Reply:
Left a note in https://github.com/astral-sh/uv/issues/3642#issuecomment-2154452853